### ISSUE TITLE
Add a parameter to simplify actions % for a single combo (Feature Request)

### ISSUE THREAD

**Issue by wishzx** [bupticybee/TexasSolver#179]
Is it possible to have a parameter that force the solver to only use fixed percentage when taking certain actions on a combo.

For example:

You could set 25% bin intervals 
This combo could only show Raise 85 25% / Fold 75%

![Screenshot 2023-09-19 172319](https://github.com/bupticybee/TexasSolver/assets/66076202/d4b0b6c8-091a-4e3a-8be6-6555a4bd00ad)

Or you could set 50% bin intervals
This combo could only show 50% Call / Raise 200 50% 

![Screenshot 2023-09-19 172303](https://github.com/bupticybee/TexasSolver/assets/66076202/f43fb924-93f1-410e-830e-d59f432bce58)

Or if you set 100% bin intervals it can only take one action for each combo.
This one would only say 100% Call

![Untitled](https://github.com/bupticybee/TexasSolver/assets/66076202/4facfb78-4d34-4d8e-93ee-01ef21ae1923)

This feature would be very useful for new students like myself because it's very hard to understand and implement the strategies when each individual combo can have many actions which are impossible to remember.

If you have an idea on how to implement this please let's have a chat and if you think this is not possible also let me know.

**Comment by bupticybee** [comment]
Yeah, it's possible and I think it's actually pretty easy to implement. Please contact me through telegram https://t.me/bupticybee .
